lizard_auth_client/views.py

```diff
@@ -721,7 +721,7 @@ def get_context_data(self, **kwargs):
         role_matrix = get_user_role_matrix_for_organisation(
             self.user, self.organisation, self.available_roles)
         self.user.role_matrix = role_matrix
-        context['user'] = self.user
+        context['logged_in_user'] = self.user
```

Comment:
Nu is alleen de key van de context variable veranderd, toch? Of anders gezegd: de getoonde naam veranderd hier toch niet door?